contents/blog/hiring-myths-and-truths.md

```diff
@@ -0,0 +1,75 @@
+---
+date: 2022-01-20
+title: Basic myths and truths about hiring that startups keep getting wrong
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - charles-cook
+featuredImage: ../images/blog/posthog-ceo-diary-blog.png
+featuredImageType: full
+category: Startups
+---
+
+These are some of the myths and truths I've learned about hiring for startups over the past decade. I hope they will help you turn recruitment into a more tolerable and less frustrating experience. 
+
+## Myth #1: Recruiters are responsible for hiring
+
+_Managers_ are responsible for hiring:
+
+- Managers are responsible for the output of their teams. 
+- Output can be increased by hiring more people. 
+- Therefore managers are responsible for hiring. 
+
+Recruiters _are_ responsible for maximizing your chances of success by running a great process, helping get a great job ad in the right places, ensuring no balls get dropped, and partnering with you to increase the size of the applicant pool. 
+
+If you are struggling to hire for a role, first ask yourself 'am I spending 20% of my time recruiting?' If the answer is yes, then go complain to your recruiter.
+
+## Myth #2: Hiring gets easier when other companies are making cuts
+
+Not really - the best people are always in demand and will have lined up 5 interviews within 24hrs of posting on LinkedIn. Often the 'cuts' that are being made are in non-core product areas, or in more manager-y type roles that are unlikely to be relevant to your early stage startup. 
+
+Hiring _should_ always be difficult because your bar should keep going up with each hire. Imagine if developing amazing new product features was easy. It would probably mean that you were building uninspiring stuff, or features that are easily replicable. 
```

Comment:
```suggestion
Not really.

The best people are always in demand and will have lined up five interviews within 24 hours of posting on LinkedIn. Often the 'cuts' that are being made are in non-core product areas, or in more manager-y type roles that are unlikely to be relevant to your early-stage startup. 

Hiring _should_ always be difficult because your bar should keep going up with each hire. Imagine if developing amazing new product features was easy. It would probably mean that you were building uninspiring stuff, or features that are easily replicable. 
```